src/components/LoanCards/Buttons/LendIncrementButton.vue

```diff
@@ -81,7 +86,7 @@ export default {
 			// IF we wanted to show this interface on loans with less than 25 remaining they would see the selector
 			const minAmount = parseFloat(this.amountLeft < 25 ? this.loan.minNoteSize : 25); // 25_hard_coded
 			// cap at 20 prices
-			const priceArray = buildPriceArray(parseFloat(this.amountLeft), minAmount).slice(0, 20);
+			const priceArray = this.enableFiveDollarsNotes ? build5DollarsPriceArray(parseFloat(this.amountLeft)).slice(0, 28) : buildPriceArray(parseFloat(this.amountLeft), minAmount).slice(0, 20); // eslint-disable-line max-len
```

Comment:
Since we have this line of code now in 3 components, it should probably be extracted to the `loanUtils` with the other new method and get a few unit tests.

Reply:
Function added.